### PR TITLE
Merge INDI simple and FULL INDI messages

### PR DESCRIPTION
**Pull request by dewagter**
- Instead of adding a FULL_INDI message, we propose to merge it with the INDI_SIMPLE.

### CODE REVIEW

File: message_definitions/v1.0/messages.xml [hunk 1]
```diff
@@ -1901,16 +1901,26 @@
...
-      <field name="angular_accel_q" type="float"/>
-      <field name="angular_accel_r" type="float"/>
-      <field name="angular_accel_ref_p" type="float"/>
-      <field name="angular_accel_ref_q" type="float"/>
-      <field name="angular_accel_ref_r" type="float"/>
-      <field name="g1_p" type="float"/>
+      <field name="body_accel_x" type="float" unit="m/s2"/><!-- setpoint -->
+      <field name="body_accel_y" type="float" unit="m/s2"/>
+      <field name="body_accel_z" type="float" unit="m/s2"/>
+      <field name="angular_rate_p" type="float" unit="rad/s" alt_unit="deg/s" alt_unit_coef="57.29578"/><!-- rate tracking -->
```
Comment (gautierhattenberger) [inline message_definitions/v1.0/messages.xml:1907]: `alt_unit_coef` is not needed for units described in https://github.com/paparazzi/pprzlink/blob/master/message_definitions/common/units.xml

Comment (dewagter) [reply message_definitions/v1.0/messages.xml]: fixed.

**Comment by dewagter** [comment]
Done

File: message_definitions/common/units.xml [hunk 2]
```diff
@@ -19,4 +19,6 @@
   <unit from="rad" to="deg" coef="57.2957795131" auto="display"/>
   <unit from="deg/s" to="rad/s" coef="0.0174532925" auto="code"/>
   <unit from="rad/s" to="deg/s" coef="57.2957795131" auto="display"/>
+  <unit from="deg/s2" to="rad/s2" coef="0.0174532925" auto="code"/>
+  <unit from="rad/s2" to="deg/s2" coef="57.2957795131" auto="display"/>
```
Comment (dewagter) [inline message_definitions/common/units.xml:23]: m/s2 is the notation for m/s^2 in the messages.xml. Is this OK?

**Review by EwoudSmeur — APPROVED** [review]
LGTM

**Pull request merged**